source/overloading.tex

```diff
@@ -3415,15 +3401,10 @@
 
 \pnum
 The allocation and deallocation functions,
-\tcode{operator}
-\tcode{new},
-\tcode{operator}
-\tcode{new[]},
-\tcode{operator}
-\tcode{delete}
-and
-\tcode{operator}
-\tcode{de\-lete\brk[]},
+\keyword{operator} \keyword{new},
+\keyword{operator} \tcode{keyword{new}[]},
+\keyword{operator} \keyword{delete}, and
+\keyword{operator} \tcode{de\-lete\brk[]},
```

Comment:
Is the lack of `\keyword` markup for this intentional? (There should be some way to combine line breaking hints with `\keyword`, but I'm not sure what.)

Reply:
\keyword puts the thing into an index. Line-breaking hints are a good way to screw up the index.

Reply:
Fixed. There isn't a linebreak where the hints are, to start with.